src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,258 @@
+import type p5 from 'p5'
+import {VisualizerDefault} from './VisualizerDefault'
+import type {VisualizerInterface} from '@/visualizers/VisualizerInterface'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import type {SequenceInterface} from '../sequences/SequenceInterface'
+
+const colorMap = new Map()
+let countClick = 0
+
+//Show description box by pressing left arrow key
```

Comment:
At some point, it would be nice if you could include some documentation for your visualizer. Look at how Glen does it in [the Show Factors Visualizer](https://github.com/numberscope/frontscope/blob/main/src/visualizers/VisualizerShowFactors.ts). Then look at how the documentation shows up on the website: https://numberscope.colorado.edu/doc/src/visualizers/VisualizerShowFactors/.

Add a comment like:

```
/** md
# Sequence Color Visualizer

Add instructions and notes for your visualizer here...
**/
```

You can add those `/** md **/` comments anywhere in your code to create your documentation. Notice how Glen splits his documentation up in Show Factors.

If you're not familiar with markdown, here's [a guide](https://www.markdownguide.org/basic-syntax/).